components/header/Buttons.tsx

```diff
@@ -26,13 +25,13 @@ function MenuButton() {
 
   return (
     <Button
-      class="btn btn-circle btn-sm btn-ghost"
+      class="btn  btn-sm btn-ghost "
```

Comment:
Adicionando um px-0 para melhorar o espaço que o logo tem no mobile e fica melhor. Quando usar o componente `Button` nao precisa usar a classe `btn` ele ja faz isso.
```suggestion
      class="btn-sm btn-ghost px-0"
```